File: app/src/main/java/com/amarnehsoft/holyquran/SplashScreen.java
package com.amarnehsoft.holyquran;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.support.annotation.Nullable;

import com.amarnehsoft.holyquran.main.MainActivity;

public class SplashScreen extends Activity{
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        new Handler().postDelayed(()->{
            startActivity(new Intent(this, MainActivity.class));
            finish();
        },300);
    }
}

File: app/src/main/java/com/amarnehsoft/holyquran/SplashScreen.kt
package com.amarnehsoft.holyquran

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler

import com.amarnehsoft.holyquran.main.MainActivity

class SplashScreen : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Handler().postDelayed({
            startActivity(MainActivity.newIntent(this))
            finish()
        }, 300)
    }
}
